### PR TITLE
registry/azure.md: fix broken syntax/links

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
It looks like this doc, which was in docker/distribution repository before, is converted to HTML tables from Markdown tables. And in the table we used to have Markdown usage for

1. links
2. backtick-escaped sections

Since the table is HTML now, those Markdown usages won't work inside HTML.

I am just noticing this and adding them back. I wish the transition was more smooth.

Signed-off-by: Ahmet Alp Balkan <ahmetalpbalkan@gmail.com>

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit 87e77ef6537f6b252370bbad3791d254d37d0626

https://deploy-preview-819--docker-docs.netlify.com

**Comment by joaofnfernandes** [comment]
Thanks for contributing. I don't see any reason for this table to be HTML and not Markdown. Do you want to convert it?

**Comment by ahmetalpbalkan** [comment]
@joaofnfernandes I don't know how we ended up here... We can keep it as is for now. Pushed just one more little fix about link rendering at the footer.

File: registry/storage-drivers/azure.md [hunk 1]
```diff
@@ -8,59 +8,12 @@ An implementation of the `storagedriver.StorageDriver` interface which uses [Mic
...
+| `accountname` | yes      | Name of the Azure Storage Account.                                                                                                                                                                                                                                  |
+| `accountkey`  | yes      | Primary or Secondary Key for the Storage Account.                                                                                                                                                                                                                   |
+| `container`   | yes      | Name of the Azure root storage container in which all registry data will be stored. Must comply the storage container name [requirements](https://docs.microsoft.com/rest/api/storageservices/fileservices/naming-and-referencing-containers--blobs--and-metadata). |
+| `realm`       | yes      | Domain name suffix for the Storage Service API endpoint. For example realm for "Azure in China" would be `core.chinacloudapi.cn` and realm for "Azure Government" would be `core.usgovcloudapi.net`. By default, this is `core.windows.net`.                        |
```
Comment (ahmetalpbalkan) [inline registry/storage-drivers/azure.md]: This "yes" should be a "no".

**Comment by joaofnfernandes** [comment]
Thanks for catching that! As soon as the tests pass I'll get that merged.

**Pull request merged**